File: java/sprint1_nonfinals/L.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Arrays;

public class L {

    private static char getExcessiveLetter(String s, String t) {
        // Ваше решение
    }

    public static void main(String[] args) throws IOException {
        try (BufferedReader reader = new BufferedReader(new InputStreamReader(System.in))) {
            String s = reader.readLine();
            String t = reader.readLine();
            System.out.println(getExcessiveLetter(s, t));

        }
    }
}

File: csharp/sprint1_nonfinals/L.cs
using System;
using System.IO;

public class L
{
    private static TextReader reader;
    private static TextWriter writer;

    private static char GetExcessiveLetter(string shorter, string longer)
    {
        // Ваше решение
    }

    public static void Main(string[] args)
    {
        reader = new StreamReader(Console.OpenStandardInput());
        writer = new StreamWriter(Console.OpenStandardOutput());

        var shorter = reader.ReadLine();
        var longer = reader.ReadLine();
        writer.WriteLine(GetExcessiveLetter(shorter, longer));

        reader.Close();
        writer.Close();
    }
}
